app/retail/templates/jtbd/earn.html

```diff
@@ -1,9 +1,315 @@
+            <path d="M177.556 71.1385C176.146 70.6422 174.627 71.5356 174.519 72.9255L173 96.9498C173 97.744 173.434 98.439 174.193 98.836L201.637 112.734C203.156 113.529 205 112.437 205 110.848L204.132 81.7609C204.132 80.9667 203.59 80.1725 202.722 79.8747L177.556 71.1385Z" fill="#DAD1FF"/>
+            <path d="M175.8 71.1005C174.5 70.6005 173.1 71.5005 173 72.9005L171.6 97.1005C171.6 97.9005 172 98.6005 172.7 99.0005L198 113C199.4 113.8 201.1 112.7 201.1 111.1L200.3 81.8005C200.3 81.0005 199.8 80.2005 199 79.9005L175.8 71.1005ZM168.9 72.7005C169.1 68.5005 173.4 65.8005 177.3 67.3005L200.4 76.2005C202.7 77.1005 204.3 79.3005 204.4 81.8005L205.2 111C205.3 115.8 200.2 118.9 196 116.6L170.7 102.6C168.6 101.5 167.4 99.2005 167.5 96.8005L168.9 72.7005Z" fill="#8C65F7"/>
+            </svg>            
+        </div>
+      </div>
+      <section id="ways-to-earn">
+        <h2 class="text-grey-500 mb-5">Ways to earn</h2>
+        <div class="row" id="accordion-group">
+          <div class="col-12 col-md-4 mr-5 ml-md-3">
+            <div class="accordion-group img-accordion" style="min-height: min(110vw, 33rem)">
+              <div class="p-4 position-absolute h-100 w-100" style="z-index: -1">
+                <img class="d-block m-auto ml-md-0" src="{% static 'v2/images/jtbd/earn-bounties.svg' %}" style="max-width: 21rem"/>
+              </div>
+              <div class="collapseOne p-4 position-absolute h-100 w-100 collapse show bg-white" data-parent="#accordion-group">
+                <img class="d-block m-auto ml-md-0" src="{% static 'v2/images/jtbd/earn-bounties.svg' %}" style="max-width: 21rem"/>
+              </div>
+              <div class="collapseTwo p-4 position-absolute h-100 w-100 collapse bg-white" data-parent="#accordion-group">
+                <img class="d-block m-auto ml-md-0" src="{% static 'v2/images/jtbd/earn-hackathons.svg' %}" style="max-width: 21rem"/>
+              </div>
+              <div class="collapseThree p-4 position-absolute h-100 w-100 collapse bg-white" data-parent="#accordion-group">
+                <img class="d-block m-auto ml-md-0" src="{% static 'v2/images/jtbd/earn-grants.svg' %}" style="max-width: 21rem"/>
+              </div>
+            </div>
+          </div>
+          <div class="col-12 col-md-5 pt-4">
+            <div class="accordion-group gc-accordion ml-md-5">
+              <h5 class="accordion" id="headingOne" data-parent="#accordion-group" data-toggle="collapse" data-target=".collapseOne">
+                <span class="accordion-title">Earn through bounties</span>
+              </h5>
+              <div class="accordion-panel collapse show font-subheader text-grey-500 collapseOne" aria-labelledby="headingOne" data-parent="#accordion-group">
+                <p>Tackle bounties for all skill levels hosted by Web3’s most exciting projects. From open security contests and new feature builds, to UX/UI design and content creation.</p>
```

Comment:
Can you swap out the `<p>` for `<h5>` on each of the `accordion-panel`'s please 👍

Reply:
done